lib/cosmos/system/target.rb

```diff
@@ -135,12 +135,9 @@ def process_file(filename)
           usage = "#{keyword} <FILENAME>"
           parser.verify_num_parameters(1, 1, usage)
           begin
```

Comment:
This only removed two lines of duplication so I'm not sure it was worth it, but I went ahead and created require_file.